### DISCUSSION TITLE
More confusion on correct flow

### DISCUSSION THREAD

**Discussion by blevine** [sventorben/keycloak-home-idp-discovery#271] [Q&A] [answered]
This is a fantastic plug-in. Thank you! 
I have a similar requirement to what was discussed in #160 in that I want a non-federated user to have to enter a password after entering his email address. I used the flow you recommended which appears to work. However, the use of an alternative flow displays the "Try another way" link which I don't want. Is there a flow that would achieve that? I realize that this is more of a Keycloak question than an issue with your plug-in, but I'd appreciate your help.

**Comment by sventorben** [comment, marked as answer]
I don't think there is a way to get rid of the "try another way" link when you have multiple alternatives to log in.
Do you use a custom theme for your login page? If so, you may simply remove the link. It is located in the `template.ftl` for the login theme: https://github.com/keycloak/keycloak/blob/22.0.5/themes/src/main/resources/theme/base/login/template.ftl#L128

**Reply by blevine** [reply]
Yes, I actually do have a custom theme so I'll do as you suggest. Thanks!
